src/Compilers/CSharp/Test/Semantic/Properties/launchSettings.json

```diff
@@ -1,12 +1,13 @@
 {
   "profiles": {
     "xUnit.net Console": {
-      "executablePath": "$(XUnitPath)",
-      "commandLineArgs": "$(XUnitArguments)"
+      "commandName": "Executable",
+      "executablePath": "C:\\Program Files\\dotnet\\dotnet.exe",
+      "commandLineArgs": "exec --depsfile E:\\code\\roslyn\\Binaries\\Debug\\UnitTests\\CSharpCompilerSemanticTest\\netcoreapp2.0\\Roslyn.Compilers.CSharp.Semantic.UnitTests.deps.json --runtimeconfig E:\\code\\roslyn\\Binaries\\Debug\\UnitTests\\CSharpCompilerSemanticTest\\netcoreapp2.0\\Roslyn.Compilers.CSharp.Semantic.UnitTests.runtimeconfig.json e:\\nuget\\xunit.runner.console\\2.3.1\\tools\\netcoreapp2.0\\xunit.console.dll E:\\code\\roslyn\\Binaries\\Debug\\UnitTests\\CSharpCompilerSemanticTest\\netcoreapp2.0\\Roslyn.Compilers.CSharp.Semantic.UnitTests.dll -method Microsoft.CodeAnalysis.CSharp.UnitTests.OutVarTests.Query_01"
```

Comment:
No

Reply:
The change to not have these settings in a .user file is so frustrating. Every single time you have to debug you break things. 😦